### ISSUE TITLE
Consider a GitHub Action for testing integration into TeSS

### ISSUE THREAD

**Issue by matuskalas** [edamontology/edamontology#859]
The TeSS's GA is https://github.com/ElixirTeSS/TeSS/blob/master/.github/workflows/test.yml

EDAM lives in TeSS here: https://github.com/ElixirTeSS/TeSS/tree/master/config/ontologies

@fbacall Would another/simpler option for us be to fork TeSS, update EDAM, and let it be tested with your CI in a draft pull request? Or do you want to avoid being spammed by PR/CI notifications? 😅

**Comment by fbacall** [comment]
Yeah, that would probably be the simplest option, and most convenient from the TeSS side.

I guess it would require some manual effort on your part though.
